### ISSUE TITLE
Substrate  Upgrade v1.0.0 Known Issues

### ISSUE THREAD

**Issue by surangap** [futureversecom/trn-seed#860]

**Comment by olexiyb** [comment]
We have upgraded 3 porchini pods on prod cluster and 3 porchini pods
So far everything running fine
The only problem I see
```
Error while dialing /dns/telemetry.rootnet.app/tcp/9443/x-parity-wss/%2Fsubmit: Custom { kind: Other, error: Timeout }    
```

**Comment by surangap** [comment]
> We have upgraded 3 porchini pods on prod cluster and 3 porchini pods So far everything running fine The only problem I see
> 
> ```
> Error while dialing /dns/telemetry.rootnet.app/tcp/9443/x-parity-wss/%2Fsubmit: Custom { kind: Other, error: Timeout }    
> ```
> 
> Here are perameters we use
> 
> ```
>       containers:
>         - args:
>             - --rpc-max-response-size=15
>             - --max-runtime-instances=8
>             - --rpc-max-connections=256
>             - --runtime-cache-size=64
>             - '--state-pruning=archive'
>             - '--blocks-pruning=archive'
>             - '--sync=full'
>             - '--name=rootscan-prod-scd1'
>             - '--chain=porcini'
>             - --rpc-external
>             - --rpc-cors=all
>             - '--telemetry-url'
>             - 'wss://telemetry.rootnet.app:9443/submit 0'
>           image: ghcr.io/futureversecom/seed:8.55.0
> ```

@olexiyb Thank you for letting us know. Added `KIS-05` above with a fix for this.

**Comment by olexiyb** [comment]
@surangap I am confused with your point
I already have `--telemetry-url=wss://telemetry.rootnet.app:9443/submit 0`
So you says that I have missing `.` and it should be `--telemetry-url=wss://telemetry.rootnet.app.:9443/submit 0`

**Comment by surangap** [comment]
> @surangap I am confused with your point I already have `--telemetry-url=wss://telemetry.rootnet.app:9443/submit 0` So you say that I have missing `.` and it should be `--telemetry-url=wss://telemetry.rootnet.app.:9443/submit 0`

@olexiyb Yes, that's correct. that extra trailing`.` will make it absolute and stop going for the DNS resolver search paths.

**Comment by olexiyb** [comment]
I still see an error
```
Error while dialing /dns/telemetry.rootnet.app./tcp/9443/x-parity-wss/%2Fsubmit: Custom { kind: Other, error: Timeout }
```
